### ISSUE TITLE
wrk is not available in apt/binary

### ISSUE THREAD

**Issue by dalisoft** [wg/wrk#436]
Hi everyone.

I use wrk sometime and want install on PopOS and Ubuntu, but it's not available in both OS.

Is there any repo i should add to get wrk?

Thanks

**Comment by dalisoft** [comment]
But i made own fork to make artifacts in GitHub actions and later i myself attach artifacts to releases for easy-to-use.

You can see latest releases from [here](https://github.com/dalisoft/wrk/releases)

**Comment by azappella** [comment]
> 
> Hi everyone.
> 
> I use wrk sometime and want install on PopOS and Ubuntu, but it's not available in both OS.
> 
> Is there any repo i should add to get wrk?
> 
> Thanks

Check this out: https://pkgs.org/search/?q=wrk

**Comment by dalisoft** [comment]
> > Hi everyone.
> > I use wrk sometime and want install on PopOS and Ubuntu, but it's not available in both OS.
> > Is there any repo i should add to get wrk?
> > Thanks
> 
> Check this out: https://ubuntu.pkgs.org/16.04/ubuntu-universe-amd64/wrk_4.0.1-2_amd64.deb.html



Thanks

**Comment by dalisoft** [comment]
Finally, Ubuntu team added **wrk** to their repos, but only available for Ubuntu 20.10+.

You can use with these commands
- `sudo add-apt-repository --yes "deb http://archive.ubuntu.com/ubuntu groovy universe"`
- `sudo apt update`
- `sudo apt install wrk`

This command worked for me, hope this helps you

**Comment by nikhilweee** [comment]
I figured that `wrk` was a part of the default repositories in Ubuntu 16.04 [[link]](https://packages.ubuntu.com/xenial/wrk), but it's no longer available on Ubuntu 18.04. So I downloaded the old deb manually, and installed using dpkg. It goes without saying, use this at your own risk.
```shell
# Download the wrk binary from Ubuntu archives
$ wget http://archive.ubuntu.com/ubuntu/pool/universe/w/wrk/wrk_4.0.1-2_amd64.deb
# Install using dpkg
$ sudo dpkg -i wrk_4.0.1-2_amd64.deb
# At this point, dpkg might complain about missing dependencies. So you use the following.
$ sudo apt install -f
```
